examples/annotated-skaffold.yaml

```diff
@@ -107,6 +107,19 @@ build:
   #   namespace: default
   #   timeout: 20m
 
+  # Docker artifacts can be built on an Azure Container Registry.
```

Comment:
I think [integration/examples/annotated-skaffold.yaml](https://github.com/GoogleContainerTools/skaffold/blob/master/integration/examples/annotated-skaffold.yaml) should be modified instead (or in addition to this one), so that when we copy over config changes we don't lose this.